app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -216,13 +216,16 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
     private ArrayList<String> reasonList;
 
     /**
-     * Height stores the height of the frame layout as soon as its intialised and updates itself on
-     * configuration change.
+     * Height stores the height of the frame layout as soon as it is initialised and updates itself on
+     * configuration changes.
      * Used to adjust aspect ratio of image when length of the image is too large.
      */
-    private int heightFrameLayuot;
+    private int frameLayoutHeight;
 
-    private int minimumHeightOfMetadata=200;
+    /**
+     * minimumHeightOfMetadataLayout in pixels.
```

Comment:
How about using the javadoc text below?
```
Minimum height of the metadata, in pixels.
Images with a very narrow aspect ratio will be reduced so that the metadata information panel always has at least this height.
```

Reply:
Thanks ;)